enhancements/validators.md

```diff
@@ -0,0 +1,256 @@
+  creation-date: 2021-12-02
+  last-updated: 2021-12-02
+  status:implemented
+  see-also:
+- https://github.com/operator-framework/enhancements/pull/98
+---
+
+# Standard definition for external validators
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implemented`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+The Validators are static checks (linters) that can scan the manifests and provide
+with low-cost valuable results to ensure the quality of the distributions which will be distributed via OLM,
+to ensure that they respect some specific criteria for vendors and/or catalogs.
+
+For example, [Openshit](https://docs.openshift.com/container-platform/4.6/operators/admin/olm-managing-custom-catalogs.html)
+catalog is composed for more than one image. To publish via each image is required to respect some specific criteria
+such as to be part of the vendor catalog.
+
+This proposal aims to provide a standard definition to reduce the complexities of keeping these rules
+maintained and consumed.
+
+## Motivation
+
+- Reduces the effort and complexities to keep the validations and rules maintained
+- Bring visibility and clarity over what are common criteria to distributed via OLM, to publish on Openshift or to distributed via certain image catalogs
+- Defining a standard definition that brings flexibility to introduce specific rules levering in the [operator-framework/API][of-api] without the need to re-implement and keep maintained what is the common use
+- A clear strategy definition about what can be introduced and supported in [operator-framework/API][of-api]
+
+### Goals
+
+- [operator-framework/API][of-api] providing and defining what is a common use for any validator such as:
+  - possible output formats
+  - common criteria, rules and suggestions to distribute solutions via OLM under Operator Framework, which are not a vendor and/or catalog specific
+  - validators interface
+  - the struct to ensure a standard for the results of the test
+- Ability to use [operator-framework/API][of-api] to build external validators with specific criteria, which are vendor or index catalog specific
+- Ability to consume many and different validators from different maintainers and sources without the need to deal with complexities related to different result structures
+- Have a standard definition that brings flexibility, allows leverage in what exists and in use without bringing extra high cost but allows us to add value with lower effort and with higher sustainability
+- Ability to use the Validators via tools, code source, ci and jobs
+- Ensure that upstream projects do not need to support and maintain specific criteria for vendors and catalogs
+
+### Non-Goals
+
+- SDK working with external validators. (_It is the scope of another PE, see [here](https://github.com/operator-framework/enhancements/pull/98)_)
```

Comment:
```suggestion
- Amending the Operator SDK so that it works with external validators. (_It is the scope of another PE, see [here](https://github.com/operator-framework/enhancements/pull/98)_)
```